src/libsyncengine/db/syncdb.cpp

```diff
@@ -1574,67 +1430,41 @@ bool SyncDb::parent(ReplicaSide side, const NodeId &nodeId, NodeId &parentNodeid
 bool SyncDb::path(ReplicaSide side, const NodeId &nodeId, SyncPath &path, bool &found) {
     const std::lock_guard<std::mutex> lock(_mutex);
 
-    std::string id = (side == ReplicaSide::Local ? SELECT_NODE_BY_NODEIDLOCAL_ID : SELECT_NODE_BY_NODEIDDRIVE_ID);
-    ASSERT(queryResetAndClearBindings(id));
-    ASSERT(queryBindValue(id, 1, nodeId));
-    if (!queryNext(id, found)) {
-        LOG_WARN(_logger, "Error getting query result: " << id.c_str()
-                                                         << (side == ReplicaSide::Local ? " - nodeIdLocal=" : " - nodeIdDrive=")
-                                                         << nodeId.c_str());
-        return false;
-    }
-    if (!found) {
-        return true;
-    }
-
-    // Fill names' vector
+    // names' vector
```

Comment:
We can remove this comment now as it only emphasizes the name of the variable.

Reply:
Done